sprucelabsai/sprucebot-skills-kit
# [SB-696] Added support for white-labelling skills

[SB-696](https://jira.sprucelabs.ai/jira/browse/SB-696)

@sprucelabsai/engineers

## Description 
We can now whitelabel specific skills by providing a value for 'WHITELABEL_STYLESHEET_URL' in the .env of a skill
## Type

## Steps to Test or Reproduce

- Add a test URL for the key 'WHITELABEL_STYLESHEET_URL' in any skill, ex: ```https://spruce-stylesheets.sprucebot.com/theme.css```
- Visit the skill and now you should notice that it is styled but leaves the other skills' styles in tact.
